pkgs/top-level/all-packages.nix

```diff
@@ -17451,6 +17451,8 @@ with pkgs;
 
   nil = callPackage ../development/tools/language-servers/nil { };
 
+  regols = callPackage ../development/tools/language-servers/regols { };
```

Comment:
There's a merge conflict in this file. Specifically for this case [RFC 140 was created](https://github.com/NixOS/rfcs/pull/140). You can switch to put files in `pkgs/by-name.` as explained in the RFC.

Reply:
Thanks @doronbehar, I've made the requested changes, and have rebuilt and tested the binary. Is this now looking OK?